ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.less

```diff
@@ -67,13 +67,32 @@
         vertical-align: top;
       }
     }
+    .is-split-button button {
```

Comment:
✏️ 	Can these styles be added directly to the `SplitButton` or `ZoomToHome`??

Reply:
I'm trying to minimize duplicate CSS, so I've found a way to do this with a mixin, but that still means it's in the less file and not the component.